doc/en/components/charts/types/bar-chart.md

```diff
@@ -2,15 +2,15 @@
 title: $PlatformShort$ Bar Charts and Graphs | $ProductName$
 _description: $PlatformShort$ Bar Charts are among the most common category chart types used to quickly compare frequency, count, total, or average of data in different categories. Try for FREE.
 _keywords: $PlatformShort$ Charts, Bar Chart, Infragistics
-mentionedTypes: ["XamDataChart"]
+mentionedTypes: ["XamDataChart", "BarSeries", `StackedBarSeries`, `Stacked100BarSeries`]
 ---
 # $PlatformShort$ Bar Chart
 
-$PlatformShort$ Bar Charts, Bar Graphs, or Horizontal Bar Charts, are among the most common category chart types used to quickly compare frequency, count, total, or average of data in different categories with data encoded by horizontal bars or equal width and differing lengths. They are ideal for showing variations in the value of an item over time. Data is represented using a collection of rectangles that extend from the left to right of the chart towards the values of data points. Like a Column Chart, or Column Graph, where bars are displayed vertically (up and down), the bar chart is rendered like the column chart, but with 90 degrees clockwise rotation and a horizontal display (left to right).
+$PlatformShort$ Bar Charts, Bar Graphs, or Horizontal Bar Charts, are among the most common category chart types used to quickly compare frequency, count, total, or average of data in different categories with data encoded by horizontal bars or equal width and differing lengths. They are ideal for showing variations in the value of an item over time. Data is represented using a collection of rectangles that extend from the left to right of the chart towards the values of data points. Like a [Column Chart](column-chart.md), or [Column Graph](column-chart.md), where bars are displayed vertically (up and down), the bar chart is rendered like the [column chart](column-chart.md), but with 90 degrees clockwise rotation and a horizontal display (left to right).
 
 ## $PlatformShort$ Bar Chart Example
 
-In this example, the bar chart is comparing movie series total world box office revenue versus the highest grossing movie in that series. The Y-Axis, or labels on the left of the chart, are displaying the string-values of the series. The X-Axis, or labels on the bottom of the chart, are displaying the value in billions of U.S. dollars.
+You can create this type of chart in the `XamDataChart` control by binding your data to a `BarSeries`, as shown in the example below:
```

Comment:
please add the old content and keep above as the last sentence as that's what I said in the task